lib/socialcast-git-extensions/cli.rb

```diff
@@ -311,6 +311,7 @@ def release
         run_cmd "git pull . #{branch}"
         run_cmd "git push origin HEAD"
         integrate_branch(base_branch, staging_branch)
+        integrate_branch(base_branch, prototype_branch) unless enforce_staging_before_release?
```

Comment:
since we're already unconditionally integrating into the staging branch, do you think we should unconditionally integrate into the prototype branch as well, for consistency/simplicity?